aion/web/elm/src/Ranking/View.elm

```diff
@@ -0,0 +1,50 @@
+module Ranking.View exposing (..)
+
+import General.Models exposing (Model)
+import Html exposing (..)
+import Msgs exposing (Msg(..))
+import Material.Table as Table
```

Comment:
I think that introducing another css library is a bad idea and we should use table from bootstrap that is already being used in a project: http://elm-bootstrap.info/table